build_tools/cmake/build_linux_benchmark_modules.sh

```diff
@@ -94,3 +94,28 @@ cd build-targets/linux-riscv
 
 "${CMAKE_BIN}" --build . --target iree-benchmark-suites-linux-riscv -- -k 0
 # --------------------------------------------------------------------------- #
+
+# --------------------------------------------------------------------------- #
+# Build for the target (linux-cuda).
+
+cd "${ROOT_DIR}"
+
+if [ -d "build-targets/linux-cuda" ]
+then
+  echo "linux-cuda directory already exists. Will use cached results there."
+else
+  echo "linux-cuda directory does not already exist. Creating a new one."
+  mkdir -p build-targets/linux-cuda
+fi
+cd build-targets/linux-cuda
+
+"${CMAKE_BIN}" -G Ninja ../.. \
```

Comment:
Why do you need to reconfigure each time? These configurations all look the same

Reply:
These benchmark suites are built in different directories. The reason we build them in different directories is because currently we don't have a good way to pack benchmark suites of different architectures if they are built under the same build dir.